src/runtime/virtcontainers/utils/utils_linux.go

```diff
@@ -89,7 +89,7 @@ func FindContextID() (*os.File, uint64, error) {
 const (
 	procMountsFile = "/proc/mounts"
 
-	fieldsPerLine = 6
+	fieldsPerLine  = 6
```

Comment:
Stray change, or vertical alignment?

Reply:
I am not sure why, but the final change follows the VS code suggestion (one space rather than vertical alignment).

Reply:
Ack.